### PR TITLE
Remove ticket link and update route

### PR DESCRIPTION
**Pull request by plaindocs**

### CODE REVIEW

**Review by mopdx — COMMENTED** [review]
The dates look accurate, and the information on the page is correct. It is ready for the world!

File: docs/conf/portland/2020/hike.rst [hunk 1]
```diff
@@ -12,14 +11,15 @@ It's rained on us the past, but we have faith it will be beautiful this year! We
 Logistics
 ---------
 
-**WARNING: This hike has been modified this year because of construction on the Wildwood trail. Instead of ending at the Zoo, we are doing an "out and back" hike to Pittock Mansion, which will start and end at the same location.**
+.. **You can get your hike ticket here**: FIXME
```
Comment (mxsasha) [inline docs/conf/portland/2020/hike.rst:14]: We should probably just delete this sentence.

**Review by mxsasha — APPROVED** [review]
I can only provide very little feedback because I don't know much about the hike :)

File: docs/conf/portland/2020/hike.rst (hunk 1 above)
Comment (plaindocs) [reply docs/conf/portland/2020/hike.rst:14]: We're gonna uncomment that later, :-D but sure.

Comment (mxsasha) [reply docs/conf/portland/2020/hike.rst]: Oh, missed the comment tag.

Comment (mxsasha) [reply docs/conf/portland/2020/hike.rst:14]: Oh, missed the comment tag.

**Pull request merged**